### PR TITLE
MIT license for tidy packages

### PR DESCRIPTION
**Pull request by topepo**
I also considered adding `use_spell_check()` but that seemed more iffy. I can update this if that's acceptable.

### CODE REVIEW

**Comment by jennybc** [comment]
Let's get @Hadley to weigh in re: license (and perhaps other stuff).

File: R/tidyverse.R [hunk 1]
```diff
@@ -73,13 +73,14 @@ create_tidy_package <- function(path,
...
   use_testthat()
-  use_gpl3_license(name)
+  use_mit_license(name)
   use_tidy_description()
 
   use_readme_rmd(open = FALSE)
   use_lifecycle_badge("experimental")
   use_cran_badge()
   use_cran_comments(open = FALSE)
+  use_code_of_conduct()
```
Comment (hadley) [inline R/tidyverse.R:83]: Isn't this already done by `use_tidy_github()`?

File: R/tidyverse.R [hunk 2]
```diff
@@ -73,13 +73,14 @@ create_tidy_package <- function(path,
 
   use_roxygen_md()
   use_testthat()
-  use_gpl3_license(name)
+  use_mit_license(name)
```
Comment (hadley) [inline R/tidyverse.R:76]: 👍

File: R/tidyverse.R (hunk 1 above)
Comment (jennybc) [reply R/tidyverse.R:83]: Yes, I've looked it up and it does. I'll remove this bit.

**Comment by jennybc** [comment]
Thanks @topepo! Note that I made a few changes, which also means I've been pushing into your `master` branch, since that's where the PR originated 😬

**Pull request merged**